src/107-Arduino-TMF8801.h

```diff
@@ -55,6 +55,8 @@ class ArduinoTMF8801 : public LengthSensorBase
 
   bool begin(uint8_t const measurement_period_ms);
 
+  void set_gpio(uint8_t num, uint8_t val);
```

Comment:
```suggestion
  void set_gpio(uint8_t const num, uint8_t const val);
```
Use const-correctness ;)
Better: Split API in 2 functions and use enum classes, i.e.
```C++
enum class GPIO : uint8_t
{
  GPIO0 = 0,
  GPIO1 = 1,
};
void set_gpio(GPIO const gpio);
void clr_gpio(GPIO const gpio);
```